### ISSUE TITLE
[Concept Entry] Sklearn ensembles

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#4932]
### Reason/inspiration (optional)

We would like a new entry on the `Ensembles` concept under Sklearn. The entry should go in a new file under `docs/content/sklearn/concepts/ensembles/ensembles.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept (if applicable)
- An `Example` section that provides an example demonstrating the concept in use (if applicable)
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use (if applicable)

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by pranavrjukanti** [comment]
@codecademy-docs Hi, I am interested in taking this issue. And just to be clear, I will have to use Markdown to resolve this issue right?

**Comment by SaviDahegaonkar** [comment]
Hey @pranavrjukanti,
Yes, you have to use the Markdown format you can get it clear after looking at the contribution guide.
You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

Thanks  & Regards,
Savi

**Comment by pranavrjukanti** [comment]
can you give me more information about what to do and better instructions
please?

**Comment by pranavrjukanti** [comment]
I need a little help to figure out exactly what I am doing.

**Comment by pranavrjukanti** [comment]
I still need a little help on what to do. I went to the Codeacademy
Contribution Guide and I'm lost. Can you please help?
Thanks a lot,
Pranav

**Comment by SaviDahegaonkar** [comment]
Hello @pranavrjukanti,
I will explain to you what you should do:
- You should write your new entry according to this path: docs/content/sklearn/concepts/ensembles/ensembles.md.
- Next, The main requirements for the entry will be an introduction section about the concept, a syntax section for that concept and example, and a code byte example for the entry related to the concept of the ensemble I  sklearn.
I think now you must have gotten the idea of how to write this PR.

**Comment by mamtawardhani** [comment]
Unassigning author due to inactivity
